moveit_plugins/moveit_ros_control_interface/moveit_ros_control_interface_plugins.xml

```diff
@@ -1,18 +1,6 @@
 <class_libraries>
-  <library path="lib/libmoveit_ros_control_interface_trajectory_plugin">
-    <class name="position_controllers/JointTrajectoryController" type="moveit_ros_control_interface::JointTrajectoryControllerAllocator" base_class_type="moveit_ros_control_interface::ControllerHandleAllocator">
-        <description></description>
-    </class>
-    <class name="velocity_controllers/JointTrajectoryController" type="moveit_ros_control_interface::JointTrajectoryControllerAllocator" base_class_type="moveit_ros_control_interface::ControllerHandleAllocator">
-        <description></description>
-    </class>
-    <class name="effort_controllers/JointTrajectoryController" type="moveit_ros_control_interface::JointTrajectoryControllerAllocator" base_class_type="moveit_ros_control_interface::ControllerHandleAllocator">
-        <description></description>
-    </class>
-    <class name="pos_vel_controllers/JointTrajectoryController" type="moveit_ros_control_interface::JointTrajectoryControllerAllocator" base_class_type="moveit_ros_control_interface::ControllerHandleAllocator">
-        <description></description>
-    </class>
-    <class name="pos_vel_acc_controllers/JointTrajectoryController" type="moveit_ros_control_interface::JointTrajectoryControllerAllocator" base_class_type="moveit_ros_control_interface::ControllerHandleAllocator">
+  <library path="moveit_ros_control_interface_trajectory_plugin">
+    <class name="joint_trajectory_controller/JointTrajectoryController" type="moveit_ros_control_interface::JointTrajectoryControllerAllocator" base_class_type="moveit_ros_control_interface::ControllerHandleAllocator">
```

Comment:
for better understanding.. All `JointTrajectoryController` types will use the same plugin in the future?

Reply:
I'm not 100% sure, but I think currently this's the only supported controller, @destogl what do you think .?

Reply:
Currently, there is only one `JointTrajectoryController` which interfaces are configurable. We are planning to add specializations into different namespaces, but the current one will always be used at the end. The specializations basically only limit parameterization possibilities.